rs/execution_environment/src/scheduler/round_schedule.rs

```diff
@@ -432,9 +436,12 @@ impl RoundSchedule {
             });
 
             total_compute_allocation_percent += compute_allocation.as_percent() as i64;
-            accumulated_priority_invariant += accumulated_priority;
-            accumulated_priority_deviation +=
-                accumulated_priority.get() * accumulated_priority.get();
+            accumulated_priority_invariant =
+                accumulated_priority_invariant.saturating_add(&accumulated_priority);
```

Comment:
While the `saturating_add()` may make sense for the variance (where you sum `accumulated_priority^2`), but we're never going to exceed `u64` with realistic numbers of canisters and rounds.